### ISSUE TITLE
div's is used as a button but lacks accessibility attributes. Add role="button", tabindex="0", and an aria-label to improve keyboard navigation and screen reader support.

### ISSUE THREAD

**Issue by Jean-LucTch** [Jean-LucTch/CrashDumpAnalyzer#43]
This div is used as a button but lacks accessibility attributes. Add role="button", tabindex="0", and an aria-label to improve keyboard navigation and screen reader support.
```suggestion
    <div class="settings-icon" onclick="toggleSettingsMenu()" role="button" tabindex="0" aria-label="Settings">
```

_Originally posted by @Copilot in https://github.com/Jean-LucTch/CrashDumpAnalyzer/pull/41#discussion_r2137311880_

This is present in Index.html - analycis.html and changelog.html
